frontend/src/App.vue

```diff
@@ -25,5 +25,5 @@ export default {
 </script>
 
 <style lang="scss">
-@import "@/core/styles/index.scss";
+@import "core/styles/index.scss";
```

Comment:
Why ? '@' is a shortcut for the root path of the project

Reply:
oh ok i got it